pkg/sdk/row_access_policies_gen.go

```diff
@@ -110,3 +111,21 @@ type RowAccessPolicyDescription struct {
 	ReturnType string
 	Body       string
 }
+
+// TODO(SNOW-1596962): Fully support VECTOR data type
+func (d *RowAccessPolicyDescription) Arguments() []map[string]any {
```

Comment:
We had so many parsing functions for arguments, don't we have one that could be used here.

Reply:
But `ParseFunctionArgumentsFromString` doesn't return arg names. Since we already have a follow-up pointed in the comment, I'll leave this for now. This signature has an odd format, I will report this internally.

Reply:
Yeah, but maybe we should add a flag to `ParseFunctionArgumentsFromString` to check for argument names or not. Maybe create a ticket in the identifier rework follow-up and reference it here, so we won't forget about it.